packages/wrangler/src/config/config-helpers.ts

```diff
@@ -23,9 +30,9 @@ export function resolveWranglerConfigPath(
 		script?: string;
 	},
 	options: { useRedirect?: boolean }
-): string | undefined {
+): ConfigPaths {
 	if (config !== undefined) {
-		return config;
+		return { userConfigPath: config, configPath: config };
```

Comment:
so they're both the same value?

Reply:
When the user provide a config path at the command line (e.g. `wrangler deploy -c my/wrangler.toml`) then we do not attemtpt to redirect and, indeed, these two are the same.